bom/pom.xml

```diff
@@ -266,6 +268,12 @@
         <artifactId>jjwt</artifactId>
         <version>${jjwt.version}</version>
       </dependency>
+      <dependency>
+        <!-- required as a (missing) transient dependency for JJWT -->
+        <groupId>javax.xml.bind</groupId>
+        <artifactId>jaxb-api</artifactId>
```

Comment:
is this no longer part of Java?

Reply:
Seems so. I need to find an explicit source for that. But that is my understanding from what I saw googling the issue. Seems like jaxb or javax stuff is no longer part of core Java.